spring/junit4-spring-base-tests/pom.xml

```diff
@@ -26,6 +26,7 @@
         <maven.deploy.skip>true</maven.deploy.skip>
         <gpg.skip>true</gpg.skip>
         <skipNexusStagingDeployMojo>true</skipNexusStagingDeployMojo>
+        <allowIncompleteProjects>true</allowIncompleteProjects>
```

Comment:
Why this is needed now?

Reply:
There was an error saying that there is no Jar so we need to change the type to POM.
Since these are test Jar modules I think it makes more sense to use this option.